src/cloud-api-adaptor/pkg/adaptor/cloud/cloud.go

```diff
@@ -35,7 +33,9 @@ import (
 
 const (
 	SrcAuthfilePath = "/root/containers/auth.json"
+	AaFilePath      = "/run/peerpod/aa.toml"
 	AuthFilePath    = "/run/peerpod/auth.json"
+	CdhFilePath     = "/run/peerpod/cdh.toml"
```

Comment:
same comment as for AA

Reply:
Makes sense to me, will make the change.